### PR TITLE
tangram, fix few out placed models, remove multiple solution levels

### PR DESCRIPTION
**Pull request by darkHarry**
This Patch removes levels with multiple solutions and fixes the positioning of a few out placed models.

### CODE REVIEW

File: src/activities/tangram/dataset.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ var dataset = [
                     },
                 ]
             },
-            {
+                {
```
Comment (petitlapin) [inline src/activities/tangram/dataset.js]: change not needed?

File: src/activities/tangram/dataset.js [hunk 2]
```diff
@@ -942,6 +942,7 @@ var dataset = [
                     },
                 ]
             },
+            /*
```
Comment (petitlapin) [inline src/activities/tangram/dataset.js]: you can directly remove them instead of commenting them

File: src/activities/tangram/Tangram.qml [hunk 3]
```diff
@@ -132,8 +132,8 @@ ActivityBase {
                     sourceSize.width: modelData.width * background.playWidth
                     sourceSize.height: modelData.height * background.playWidth
                     z: index
-                    rotation: modelData.flipping ? 360 - modelData.rotation : modelData.rotation
-                    mirror: modelData.flipping ? true : false
+                    rotation: modelData.rotation
```
Comment (petitlapin) [inline src/activities/tangram/Tangram.qml]: do you have an example of level where it applies? I'm not sure to understand the change

File: src/activities/tangram/Tangram.qml [hunk 4]
```diff
@@ -132,8 +132,8 @@ ActivityBase {
...
+                    mirror: modelData.flipping ? true : false
```
Comment (petitlapin) [inline src/activities/tangram/Tangram.qml]: can you replace the character with a normal space? I'm not sure on how it went there on the first place but let's use normal spaces.

File: src/activities/tangram/Tangram.qml (hunk 3 above)
Comment (darkHarry) [reply src/activities/tangram/Tangram.qml]: 11 levels created this problem with parallelogram. *screenshot*
The rotation of modelTan was out of place and the solution was being accepted.

File: src/activities/tangram/Tangram.qml (hunk 4 above)
Comment (darkHarry) [reply src/activities/tangram/Tangram.qml]: it comes whenever i change the line above it in qt creator. Used other editor and didn't happen. :/

**Comment by Animtim** [comment]
Hi, looks like you didn't do exacly what was asked... except for level 11 where it doesn't look the same after your qml change, I don't understand why you had to change the dataset like that?

**Comment by petitlapin** [comment]
committed in https://commits.kde.org/gcompris/0bdda9aa7da4198726ca77f3339bc661873b1e5f

Thank you,

Johnny

**Pull request closed without merging**